app/assets/javascripts/discourse/tests/integration/components/keyboard-shortcuts-test.js

```diff
@@ -117,22 +118,22 @@ Object.keys(functionBindings).forEach((func) => {
 });
 
 test("selectDown calls _moveSelection with 1", (assert) => {
-  var stub = sandbox.stub(KeyboardShortcuts, "_moveSelection");
+  var stub = sinon.stub(KeyboardShortcuts, "_moveSelection");
```

Comment:
I see you replaced var with let in other files in this PR but not those in this one. (except a single instance at the top)

Do you want to update this PR or do a second one that updates all  vars across all tests?